test/format/format_inline.h

```diff
@@ -320,12 +320,19 @@ static inline void
 wt_wrap_begin_transaction(WT_SESSION *session, const char *config)
 {
     WT_DECL_RET;
+    char transaction_config[MAX_FORMAT_PATH];
```

Comment:
The comment on `MAX_FORMAT_PATH` leads me to think it has to do with file system paths. Is it appropriate for the largest possible test configuration?

Reply:
I can definitely change this to some other arbitrary value if that makes more sense.

Reply:
Fixed in [4de96ae](https://github.com/wiredtiger/wiredtiger/pull/9357/commits/4de96ae50ca00ba0bde584aabe445769f4f6ac27)